runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -8135,10 +8137,11 @@ reservationLockEnter(TR::Node *node, int32_t lwOffset, TR::Register *objectClass
       // #IF 64-Bit && JAVA_VERSION >=19
       //    incrementOwnedMonitorCountLabel:
       //    AGSI #ownedMonitorCountOffset(J9VMThread), 1
+      // #ENDIF
       // BRC   returnLabel
       // callHelper:
       // BRASL R14, jitMonitorEntry
-      //returnLabel:
+      // ßreturnLabel:
```

Comment:
I'm not sure what that character before `returnLabel` is, but I don't think it belongs here.